examples/GeoTiffExample.js

```diff
@@ -26,7 +26,7 @@ requirejs(['./WorldWindShim',
             wwd.addLayer(layers[l].layer);
         }
 
-        var resourcesUrl = "http://worldwindserver.net/webworldwind/data/black_sea_rgb.tif";
+        var resourcesUrl = "https://worldwind.arc.nasa.gov/web/examples/data/black_sea_rgb.tif";
```

Comment:
I'm sure it's just an FYI, but a log message "Ignored GeoTiff key: 3076" is generated by this example. Also, I will create an issue to add a loading progress bar to this example as the browser freezes for several seconds while the TIFF is downloading.